index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>Portfolio</title>
+	<link rel="stylesheet" href="style.css"/>
+	<link rel="preconnect" href="https://fonts.gstatic.com"/>
+	<meta name="viewport" content="width=device-width, initial-scale=1"/>
+	<link href="https://fonts.googleapis.com/css2?family=Nixie+One&display=swap" rel="stylesheet"/>
+</head>
+<body class="background">
+	<header>
+		<nav>
+			<a href="#home">home.</a>
+			<a href="#projects">projects.</a>
+			<a href="#about">about.</a>
+			<a href="#contact">contact.</a>
+		</nav>
+	</header>
+	<main id="home">
+		<div class="top" id="fade">
+		<h1>maria</h1>
+		<h1>gusova.</h1>
+	</div>
+	<div class="bottom" id="fade">
+		<h2>software</h2>
+		<h2>developer.</h2>
+	</div>
+	</main>
+	<section id="projects" class="projects">
+		<h3>projects.</h3>
+		<div class="grid">
+			<div class="project">
+				<a href="https://github.com/firefox-devtools/profiler"><img src="profiler.png" alt="firefox profiler"/><p>Firefox Profiler</p></a>
+			</div>
+			<div class="project">
+				<a href="https://mariagus.github.io/de-tough/"><img src="detough.png" alt="DE-Tough"/><p>DE-Tough LTD</p></a>
+			</div>
+			<div class="project">
+				<a href="https://mariagus.github.io/grumpygit/"><img src="grumpygit.png" alt="Grumpy Git"/><p>Grumpy Git</p></a>
+			</div>
+			<div class="project">
+				<a href="https://mariagus.github.io/pomodoro-clock/"><img src="pomodoro.png" alt="Pomodoro Clock"/><p>Pomodoro Clock</p></a>
+			</div>
+		</div>
+	</section>
+	<section id="about">
+		<div class="about-me">
+		<h3>about me.</h3>
+			<img src="me.jpeg" alt="me" />	
+			<div class="text">	
+				 <p>
+		        I'm Maria, a software developer based in Bristol, UK. <br/> I discovered my passion for programming in 2019, when I realised how much I enjoy problem-solving. My initial focus was on gaining a strong foundation in JavaScript and working through algorithms, before moving on to building personal projects and contributing to open source.</p>
+		        <p>Technologies I have used:</p>
+				<ul>
+		        	<li>JavaScript (ES6+)</li>
+		        	<li>HTML & CSS</li>
+		        	<li>React</li>
+		        	<li>Git</li>
+		        	<li>Redux</li>
+		        	<li>Flow</li>
+		        	<li>Jest</li>
+		       	 	<li>Express</li>
+		        	<li>MongoDB</li>
+		        	<li>Mongoose</li>
+			</ul>
+		</div>
+		        </div>
+	</section>
+	<section id="contact" class="contact">
+		<a class="contact-box" href="mailto:masha.gu@gmail.com"><h3 class="contact-font"><span>don't</span> get in touch.</h3></a>
+		<a class="git" href="https://github.com/mariagus"><img src="github.png" alt="github icon"/></a>
```

Comment:
I would suggest making this link open in a new tab, just to stop people from leaving your site if they click it! You can do that by giving the a tag a `target="_blank"` attribute